bin/deprecateAdv.awk

```diff
@@ -11,6 +11,9 @@ match($0, /.*(UNIT_.*)_PREREQ.*/, s) {
     found=1
     }
 (found++) == 2 && OA {
-    printf("Obsolete:\n<L:DATABASE_ADVANCES,%s>%s<e>\n", OA, OA)
+    oa=OA
+    gsub(/ADVANCE_/, "", oa)
+    gsub(/_/, " ", oa)
+    printf("Obsolete:\n<L:DATABASE_ADVANCES,%s>%s<e>\n", OA, tolower(oa))
```

Comment:
The initials should be upper case, in fact you want to take this from the string database, anyway. Since you could name these things whatever you like. And in a localized version those don't fit anyway.

In fact, we could add a default value which would be loaded from the string database. Something like you have in Wikipeda, where you could append the link with an s for instance and it is still part of the link.

Reply:
I see, you mean like using
https://github.com/civctp2/civctp2/blob/11b9a024f4e0b5ace7011f6e53a49f5f7bb83651/ctp2_data/english/gamedata/gl_str.txt#L83
for the English version and otherwise the corresponding language file.

Reply:
Done with https://github.com/civctp2/civctp2/pull/319/commits/c46cea6ae07a7cf67798f7d28351af965b4a28b0.